src/components/Home.js

```diff
@@ -8,6 +8,11 @@ import useFetch from "../Hooks/Use-fetch";
 const Home = () => {
 
     const { loading } = useFetch('http://localhost:8000/courses');
+
+    const secret = process.env.REACT_API
+
+  console.log(secret)
+  
```

Comment:
@Craftsman8, Kindly check the WhatsApp group for the message I sent yesterday regarding this as this exact setup, used during the video, has an issue. I already gave the correction.